### PR TITLE
Enable debug log level on the operator for E2E tests

### PR DESCRIPTION
**Pull request by sebgl**
This should make it easier to debug failing E2E tests if the operator
gives us a bit more logs.

### CODE REVIEW

**Comment by charith-elastic** [comment]
Should we do this on a case-by-case basis (by making it configurable). This change would also affect matrix jobs like the stack versions job where the logs are already very large and hard to parse.

**Comment by sebgl** [comment]
Oh you're right @charith-elastic I did not think about it.

Now that we have tgz archives updated by Jenkins with the full logs I'm wondering whether we should remove the operator logs entirely from the Jenkins job output?
If we need the logs, we can look into the tgz archive. WDYT?

**Comment by charith-elastic** [comment]
I wasn't thinking clearly. This change would only change the operator log level -- which doesn't get captured by Jenkins anyway. It should be fine to go ahead with this PR. Sorry!

**Comment by sebgl** [comment]
@charith-elastic oh you're right we're only grabbing the tests logs + k8s events, not the operator logs.
I wasn't thinking clearly either 😄
